packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -110,7 +110,9 @@ const metadata: Meta<TableArgs> = {
             id-field-name="${x => dataSetIdFieldNames[x.data]}"
             data-unused="${x => x.tableRef.setData(dataSets[x.data])}"
         >
```

Comment:
What's your vision for documenting column APIs? We have #975 to track documenting specific column types but I'm not clear what the plan is to document APIs like this one for providing content and also all the size/sorting/grouping config that's coming soon.

This doesn't necessarily have to be fixed in this PR but we should collect use cases and track them in an issue.

Reply:
I will create an issue related to this as a follow-up to submitting this PR.